test/controllers/ProcessReviewControllerSpec.scala

```diff
@@ -698,100 +656,94 @@ class ProcessReviewControllerSpec extends WordSpec with Matchers with ScalaFutur
 
     "the request is valid" should {
 
-      trait ValidTest extends Test {
+      "correctly return an OK response" in new Test {
+
         MockReviewService
           .factCheckComplete(validProcessIdForReview, statusChangeInfo)
-          .returns(Future.successful(Right(())))
+          .returns(Future.successful(Right(approvalProcessCompleted)))
 
         lazy val request: FakeRequest[JsValue] = FakeRequest().withBody(statusChangeJson)
-      }
-
-      "return a NO_CONTENT response" in new ValidTest {
         private val result = controller.approvalFactCheckComplete(validProcessIdForReview)(request)
-        status(result) shouldBe NO_CONTENT
+        status(result) shouldBe OK
+        contentType(result) shouldBe Some("application/json")
       }
 
-      "return no content" in new ValidTest {
+      "correctly return a BAD_REQUEST if the response contains an invalid process" in new Test {
+
+        MockReviewService
+          .factCheckComplete(validProcessIdForReview, statusChangeInfo)
+          .returns(Future.successful(Right(approvalProcess)))
```

Comment:
Approval process contains an invalid process; perhaps it could be better named.

Reply:
updated, will be in next commit